docs/layers/base-layer.md

```diff
@@ -201,45 +183,18 @@ can be refactored to avoid the need.
 
 ##### `numInstances` (Number, optional)
 
-deck.gl is often able to autodetect the number of objects in your `data` prop,
-however in special situations it can be useful to control this directly.
+deck.gl automatically derives the number of drawing instances from the `data` prop by
+counting the number of objects in `data`. However, the developer might want to
+maunally override it using this prop.
 
 ##### `updateTriggers`
 
-This prop expects an object with keys matching attribute names.
-Each attributeName will contain an object with keys representing property names.
-If any property mentioned in the updateProps has changed, the corresponding
-attribute will be invalidated.
-
-This way, if an attribute update depends on properties other than data,
-the layer can ensure that the attribute gets autoupdated when those props
-change.
-
-The value of `updateTriggers` is a map with fields corresponding to
-attribute names (or `all`). Each field has a value which is an object,
-it can contain any amount of data. The data for each field is compared
-shallowly, and if a change is detected, the attribute is invalidated
-(all attributes are invalidated if the `all` key is used.)
-
-Note: updateTriggers are ignored by the normal shallow comparison, so it is
-ok for the app to mint a new object on every render.
-
-##### "attributes" properties
```

Comment:
So `attributes` is the prop that we need for the PointCloudLayer (enabling binary PLY loader to just pass in buffers, defeating attribute generation).

It is implemented in attribute manager but not tested. On one hand it makes sense to not include the docs. But in that case we should probably create a new PR that adds it back for next release so we don't have to rewrite this.